includes/reader-revenue/woocommerce/class-woocommerce-connection.php

```diff
@@ -522,6 +523,95 @@ public static function send_customizable_receipt_email( $enable, $order, $class
 		return false;
 	}
 
+	/**
+	 * Send the customizable cancellation email in addition to WooCommerce Subscription's default.
+	 * We still want to allow WCS to send its cancellation email since this targets the store admin.
+	 *
+	 * @param bool            $enable        Whether to send the cancellation email.
+	 * @param WC_Subscription $subscription  The order object for the cancellation email.
+	 * @param WC_Email        $class         Instance of the WC_Email class.
+	 *
+	 * @return bool
+	 */
+	public static function send_customizable_cancellation_email( $enable, $subscription, $class ) {
+		// If we don't have a valid subscription, or the customizable email isn't enabled, bail.
+		if ( ! is_a( $subscription, 'WC_Subscription' ) || ! Emails::can_send_email( Reader_Revenue_Emails::EMAIL_TYPES['CANCELLATION'] ) ) {
+			return $enable;
+		}
+
+		$frequencies = [
+			'month' => __( 'Monthly', 'newspack-plugin' ),
+			'year'  => __( 'Yearly', 'newspack-plugin' ),
+		];
+		$product_map = [];
+		foreach ( $frequencies as $frequency => $label ) {
+			$product_id = Donations::get_donation_product( $frequency );
+			if ( $product_id ) {
+				$product_map[ $product_id ] = $label;
+			}
+		}
+
+		$items = $subscription->get_items();
+		$item  = array_shift( $items );
+		$is_donation = Donations::is_donation_product( $item->get_product_id() );
```

Comment:
When deleting and account which has active subscriptions, this method will throw with 

```
PHP Fatal error:  Uncaught Error: Call to a member function get_product_id() on null
```

because the `$item` is `null` at this point.

Reply:
Thanks for catching this! I'll get a fix up for this today 👍